### PR TITLE
added mutation for Persons auth method reset

### PR DESCRIPTION
**Pull request by PavelVesnin**
#3037

### CODE REVIEW

File: apps/core/lib/core/persons/persons.ex [hunk 1]
```diff
@@ -83,19 +84,29 @@ defmodule Core.Persons do
   end
 
   def get_by_id(id) do
-    with {:ok, person} <- @rpc_worker.run("mpi", Core.Rpc, :get_person_by_id, [id]) do
```
Comment (progsmile) [inline apps/core/lib/core/persons/persons.ex]: Don't use alias here, that's mpi module

File: apps/core/lib/core/persons/persons.ex [hunk 2]
```diff
@@ -83,19 +84,29 @@ defmodule Core.Persons do
...
+    with {:ok, person} <- @rpc_worker.run("mpi", Rpc, :get_person_by_id, [id]) do
       {:ok, person}
     else
       _ -> {:error, {:not_found, "Person not found"}}
     end
   end
 
   def list(params, [{_direction, _attribute}] = order_by, {_limit, _offset} = opts) do
-    with {:ok, persons} <- @rpc_worker.run("mpi", Core.Rpc, :search_persons, [params, order_by, opts]) do
+    with {:ok, persons} <- @rpc_worker.run("mpi", Rpc, :search_persons, [params, order_by, opts]) do
```
Comment (progsmile) [inline apps/core/lib/core/persons/persons.ex]: same

File: apps/core/lib/core/persons/persons.ex [hunk 3]
```diff
@@ -83,19 +84,29 @@ defmodule Core.Persons do
...
       {:ok, persons}
     end
   end
 
+  def reset_person_auth_method(id, headers) do
+    user_id = get_consumer_id(headers)
+
+    with {:ok, person} <- @rpc_worker.run("mpi", Rpc, :reset_auth_method, [id, user_id]) do
```
Comment (progsmile) [inline apps/core/lib/core/persons/persons.ex]: same

File: apps/graphql/test/web/resolvers/person_resolver_test.exs [hunk 4]
```diff
@@ -239,4 +254,44 @@ defmodule GraphQLWeb.PersonResolverTest do
       assert "NOT_FOUND" == error["extensions"]["code"]
     end
   end
+
+  describe "reset authentication method to NA" do
+    test "success", %{conn: conn} do
+      person = build(:person, authentication_methods: [%{"type" => "NA"}])
```
Comment (progsmile) [inline apps/graphql/test/web/resolvers/person_resolver_test.exs]: use `build(:mpi_person ...)`

Comment (PavelVesnin) [reply apps/graphql/test/web/resolvers/person_resolver_test.exs]: what's the difference?

Comment (PavelVesnin) [reply apps/graphql/test/web/resolvers/person_resolver_test.exs]: @progsmile done

**Pull request merged**